### DISCUSSION TITLE
How do I properly set up python for Keras.NET?

### DISCUSSION THREAD

**Discussion by CodeRabbit957** [SciSharp/Keras.NET#244] [Q&A]
When I try to create my model, it says: "Unable to load DLL 'python3.8m' or one of its dependencies: The specified module could not be found.". 
I'm not sure what to do here - if I need to get the program to check that python is installed and install it if not, then how do I do that? Example code would be helpful.

Thanks in advance.

**Comment by carl2099** [comment]
I think downloading and installing python3.8 is the most direct solution. You need to add python 3.8 to your PATH. You can find related tutorials from the Internet. At the same time, you need to ensure that python38 instead of python38/Scripts is at the top
